src/services/game.service.ts

```diff
@@ -34,38 +33,42 @@ class GameService {
     let word;
     do {
       word = vocabulary.words[Math.floor(Math.random() * vocabulary.words.length)];
-    } while (await this.wordWasUsedInGame(word, game._id!))
+    } while (await this.wordWasUsedInGame(word, game._id!));
     return word;
   }
 
   private async wordWasUsedInGame(word: string, gameId: string): Promise<boolean> {
     const rounds = await roundService.getAllByGameId(gameId);
-    return rounds.some(round => round.words.map(obj => obj.word).includes(word));
+    return rounds.some((round) => round.words.map((obj) => obj.word).includes(word));
   }
 
-  async start(id: string) {
+  async start(id: string): Promise<{ msg: string; chatId: string }> {
     await gameRepository.start(id);
     const game = await this.getById(id);
-    const roundId = await this.createRound(game);
+    const { roundId, chatId } = await this.createRound(game);
     game.rounds.push(roundId);
     await this.update(game);
-    //TODO provide chat
+    return {
+      msg: `proceed to chat with chatId:${chatId}. Link: http://localhost:3000/api/v1/chats/${chatId}/view`,
+      chatId,
+    };
   }
 
-  private async createRound(game: IGame): Promise<string> {
+  private async createRound(game: IGame): Promise<{ roundId: string; chatId: string }> {
+    const chatId = await chatService.create();
     const roundId = await roundService.create({
       teamId: game.currentTeam,
       hostId: await this.getHostIdFromTeam(game.currentTeam, game._id!),
-      chatId: await chatService.create(),
+      chatId: chatId,
       finishedAt: this.getFinishDate(new Date(), game.options.roundTime),
-      currentWord: await this.getRandomWord(game._id!)
-    } as IRoundCreateSchema)
-    return roundId;
+      currentWord: await this.getRandomWord(game._id!),
+    } as IRoundCreateSchema);
+    return { roundId, chatId };
   }
 
   async handleFinishedRound(id: string, round: IRound) {
     const game = await this.getById(id);
```

Comment:
The update of the game state is dependent on multiple asynchronous operations. If one fails, it could lead to inconsistent game state.